android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -103,7 +104,6 @@
 import com.facebook.react.bridge.ReadableMap;
 import com.facebook.react.uimanager.ThemedReactContext;
 import com.google.ads.interactivemedia.v3.api.AdEvent;
-import com.google.android.exoplayer2.ext.ima.ImaAdsLoader;
```

Comment:
Yes, but no :)
In fact these files are "stubs".
As there is no conditional compilation in Java, we added a stub to be able to avoid including google IMA package.
These files are here to stub google package. There is maybe a better way to do that instead of this stub, but it works like that for now ...

Reply:
notice also that adding IMA libs in the package imply to configure it, mainly when you push your app on store (AD_ID). This is not expected to handle that by default

Reply:
Oh ok! Sorry didn't think through it! 😅 I was using ads and forgot that we could actually not use them at all and so these imports would break, thanks for the explanation! 🙂